pkg/templates/templates/create/template.buildkit.Dockerfile

```diff
@@ -10,7 +10,7 @@
 
 # You can control where the mixin's Dockerfile lines are inserted into this file by moving the "# PORTER_*" tokens
 # another location in this file. If you remove a token, its content is appended to the end of the Dockerfile.
-FROM debian:stretch-slim
+FROM --platform=linux/amd64 debian:stretch-slim
```

Comment:
This is the absolute best place to provide context and instructions since everyone will have the file even if they never find their way to the right page on the website.

Let's add a comment that explains that we are fixed by linux/amd64 _by default_ and this can be changed by editing the --platform flag.

```
# You can control where the mixin's Dockerfile lines are inserted into this file by moving the "# PORTER_*" tokens
# another location in this file. If you remove a token, its content is appended to the end of the Dockerfile.

# Porter targets linux/amd64 by default. Change the --platform flag to target a different platform.
FROM --platform=linux/amd64 debian:stretch-slim
```